labs/203-custom-userspace-instrumentation-in-c/code/makefile

```diff
@@ -0,0 +1,11 @@
+EXECS=ring
+MPICC?=mpicc
+
+all: ${EXECS}
+
+ring: ring.c
+	gcc -I. -c ring_tp.c
+	${MPICC} -o ring ring_tp.o ring.c -llttng-ust -ldl
```

Comment:
You can use a single line to compile both c files :

    ${MPICC} -o ring -I. ring_tp.c ring.c -llttng-ust -ldl